### ISSUE TITLE
Add placeholder image

### ISSUE THREAD

**Issue by TheWeirdSquid** [TheWeirdSquid/cufries#12]
Currently, if a food item does not have an image, then the default "image missing" icon appears as well as the alt text, which simply reads "image". Frankly, this looks terrible. Adding a placeholder image for items which do not have images would make the site look better and fix some sizing issues on Microsoft Edge, who ignores image size rules if the image does not load.

In order to do this, a handler could be placed on the image element to detect when load fails (onError), which then replaces it's src tag with the placeholder image url.

**Comment by TheWeirdSquid** [comment]
The preview image does work, but it only loads in after a couple seconds, which kind of defeats the point. The component needs to be able to tell ahead of time which image it will use, then render it with that in the first place, rather than resetting the component after it has loaded.

**Comment by TheWeirdSquid** [comment]
Alright. Unfortunately next likes being difficult, so I had to go with a roundabout approach. Instead of getting the images through the server directly, I created an endpoint (/api/getimage) that takes a name query parameter. It will then look for the image for that food. If it finds it, it returns the image. If not, it returns the placeholder image. As far as the client is aware, that's the image it was supposed to have the whole time, so there's no additionally loading or flickering. It certainly increases compute costs though, so we might want to find another solution at some point. For now I'm going to close the issue and mark it as done.
